docker-compose.yml

```diff
@@ -1,13 +1,14 @@
 version: '3'
 services:
   app:
-    build:
-      context: .
-      dockerfile: Dockerfile
+    build: .
+    restart: always
     depends_on:
       - db
     ports:
       - 8000:8000
+    volumes:
+      - ./grasa_event_locator:/app/grasa_event_locator:z
```

Comment:
Will this work on Windows?

Reply:
@nlMeminger I am fairly confident. I used this syntax in the Django project I managed over the summer for persistent volumes. This worked for most folks who set up a development environment of our project during the application period. I give this 90% confidence.